Doc/library/pathlib.rst

```diff
@@ -1147,7 +1147,10 @@ call fails (for example because the path doesn't exist).
 
    Rename this file or directory to the given *target*, and return a new Path
    instance pointing to *target*.  On Unix, if *target* exists and is a file,
-   it will be replaced silently if the user has permission.
+   it will be replaced silently if the user has permission. The operation may
+   fail on some Unix flavors if *src* and *dst* are on different filesystems.
+   If successful, the renaming will be an atomic operation (this is a POSIX requirement).
```

Comment:
I don't want to make this promise as part of the API as there is no explicit guarantee the OS is POSIX-compliant.

```suggestion
```

Reply:
So, in order to create a file in an atomic way, do I have to use `os.rename` (which is used in `Path.rename`)?

Reply:
Good point as [`os.rename()`](https://docs.python.org/3/library/os.html#os.rename) makes that promise. 🤔 I guess it's a question of whether we think it's reasonable to assume we will rely on `os.rename()` going forward. At which point it might sense to link to the `os.rename()` docs and say those guarantees hold rather than duplicate them here.

Reply:
I have linked it (and deleted my previous commit).